wp-content/themes/csisnuclearnetwork/assets/_scss/layout/_footer.scss

```diff
@@ -126,6 +126,12 @@
     }
   }
 
+  &__x-icon {
+    position: relative;
+    top: rem(6);
+    left: rem(6);
+  }
+
```

Comment:
To be consistent, let's remove this and adjust the position in the viewbox in the icons.svg file.